README.md

```diff
@@ -1,3 +1,29 @@
+````
+-----------------------------          _______  _______  _______  _______ ------------------------------
+-----------------------------|\     /|(  ____ \(  ___  )(  ____ \(  ____ \------------------------------
+-----------------------------| )   ( || (    \/| (   ) || (    \/| (    \/------------------------------
+-----------------------------| |   | || (__    | |   | || (_____ | (_____ ------------------------------
+-----------------------------( (   ) )|  __)   | |   | |(_____  )(_____  )------------------------------
+----------------------------- \ \_/ / | (      | |   | |      ) |      ) |------------------------------
+-----------------------------  \   /  | )      | (___) |/\____) |/\____) |------------------------------
+-----------------------------   \_/   |/       (_______)\_______)\_______)------------------------------
+                                   Vanderbilt Free & Open Source Society
+````                                             
+
+<p align="center"><strong>Welcome to Vandy's Open Source One Stop Shop</strong></p>
```

Comment:
Idk if we need more of a slogan but not totally sold on this

Reply:
changed to "Welcome to VFOSS"